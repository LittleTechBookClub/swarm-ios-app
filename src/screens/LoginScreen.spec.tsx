/**
 * @jest-environment jsdom
 */

import React from 'react';
import { render } from '@testing-library/react-native';
import LoginScreen from './LoginScreen';

describe('LoginScreen', () => {
  it('should render the Login Screen', () => {
    const tree = render(<LoginScreen />);
    expect(tree).toMatchSnapshot();
  });
});
