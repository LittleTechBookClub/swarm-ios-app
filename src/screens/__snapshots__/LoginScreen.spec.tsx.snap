// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreen should render the Login Screen 1`] = `
<div>
  LoginScreen
</div>
`;
